07LessonSeven/homework/index.html

```diff
@@ -4,32 +4,36 @@
   <title>Profession Portal</title>
   <link rel="stylesheet" href="css/style.css">
   </head>
-
   <body>
-    <h1>Profession Portal</h1>
-    <nav>
-      <ul>
-        <li><a target="_blank" href="#">Home</a></li>
-        <li><a target="_blank" href="#">Companies</a></li>
-        <li><a target="_blank" href="#">Positions</a></li>
```

Comment:
Don't use the `target="_blank"` attribute when just setting the `href="#"`. You use the target property only when you have an actual `href` value that takes the user to another page and you want that page to be opened in a new browser window or tab.